ducktape/tests/runner_client.py

```diff
@@ -176,11 +179,26 @@ def run(self):
             self.test_context = None
             self.test = None
 
-    def _check_cluster_utilization(self):
+    def _check_cluster_utilization(self, result, summary):
+        """Pass through results and summary and modify the values if
```

Comment:
This could probably be a bit clearer -- it's not obvious from the comment what "modify the values mean" and the subsequent sentence makes it sound like it'll only ever warn.

Reply:
Thanks for the feedback hopefully it is clearer now